src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -905,6 +909,7 @@ static bool isSupportedReferenceTypeValue(BoundExpression value)
                     case BoundKind.ObjectCreationExpression:
                     case BoundKind.AnonymousObjectCreationExpression:
                     case BoundKind.DynamicObjectCreationExpression:
+                    case BoundKind.NewT:
```

Comment:
>NewT [](start = 35, length = 4)

Should this depend on the `T` (constrained or not, ...)?

Reply:
I believe we should allow all type parameters.

---
In reply to: [265721808](https://github.com/dotnet/roslyn/pull/34087#discussion_r265721808) [](ancestors = 265721808)